docs/installation/using-multiple-databases.md

```diff
@@ -24,13 +24,13 @@ In the example below, the `mysql` driver is used, but you can use any driver you
     'tenant' => [
         'driver' => 'mysql',
         'database' => null,
-        // other options such as host, username, password, ...
```

Comment:
I think is better to add `'username' => null,`, etc, instead of changing the comment.

Reply:
Understood. Details added on the next commit.